shared/src/main/java/org/odk/collect/shared/PathUtils.kt

```diff
@@ -1,26 +1,12 @@
 package org.odk.collect.shared
 
-import java.io.File
-
 object PathUtils {
 
     @JvmStatic
     fun getRelativeFilePath(dirPath: String, filePath: String): String {
         return if (filePath.startsWith(dirPath)) filePath.substring(dirPath.length + 1) else filePath
     }
 
-    @JvmStatic
-    fun getAbsoluteFilePath(dirPath: String, filePath: String): String {
```

Comment:
I had to move this whole method to the `androidshared` module because the `shared` one is a pure java/kotlin module and I can't use Timber or Analytics there. I've tried `SLF4J` to no avail. I think we should file an issue to figure out how to log things in such modules.